tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -1,12 +1,14 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "# Homology Modelling Tutorial\n",
-    "Max Epstein and Phil Biggin (based on previous work from Phill Stansfeld)\n",
+    "Max Epstein, Bjarne Feddersen and Phil Biggin (based on previous work from Phill Stansfeld)\n",
```

Comment:
Should we move these to acknowledgment files like we do for the python & MD tutorials?

Reply:
Can we leave at the top for now.